notebooks/04-Creating_ASDF_Files.ipynb

```diff
@@ -0,0 +1,753 @@
+{
```

Comment:
<p>I don't think the mode keyword is needed for read mode.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p><code>mode</code> must be used if you want to specify the open method. Well the issue is more complicated, there is another option before <code>mode</code> in the interface, so naively specifying without the keyword is dangerous.</p>